templates/pages/docs.html

```diff
@@ -33,12 +33,12 @@ <h2>Overview</h2>
       <li><strong>Step 6.</strong> Share your results! The link to every results page is static and will never change. Send them around.</li>
       </ul>
 
-      <p> Throughout this process, if you have a question about how to use TaxBrain or interpret the results, if you want to make a suggestion for making the interface or underlying models better, or if you discover a bug, please <a href = '//list.ospc.org/mailman/listinfo/users_list.ospc.org' target="_blank">join our mailing list</a> and send a message or <a href = '//ospc.org/hello' target="_blank">leave a note for the OSPC team</a>.</p>
+      <p> Throughout this process, if you have a question about how to use TaxBrain or interpret the results, if you want to make a suggestion for making the interface or underlying models better, or if you discover a bug, please <a href = '//list.ospc.org/mailman/listinfo/users_list.ospc.org' target="_blank">join our mailing list</a> and send a message or <a href = '//www.ospc.org/hello' target="_blank">leave a note for the OSPC team</a>.</p>
```

Comment:
@hdoupe won't this line fix #737?

Reply:
I just tested it and it didn't seem to work correctly.  I'll take another look though.

Reply:
Strange. Both links are working on my local.

Reply:
Just checked again, and it worked.  Sorry about that.